packages/search-adapter/package.json

```diff
@@ -1,10 +1,32 @@
 {
   "name": "@empathyco/x-adapter",
   "version": "6.1.0",
-  "description": "A search client for the Empathy search API ",
+  "description": "Empathy search types",
```

Comment:
Just Empathy  X types?

Reply:
The fuck did I do there, this is the adapter, not the search types